moved/src/types/state.rs

```diff
@@ -370,6 +375,9 @@ fn compute_from(tx: &OpTxEnvelope) -> Result<Address, alloy::primitives::Signatu
     }
 }
 
+#[derive(Debug)]
+pub struct TransactionResponse(pub op_alloy::rpc_types::Transaction);
```

Comment:
Why isn't it similar to TransactionReceipt, like:
```rust
pub type TransactionResponse = op_alloy::rpc_types::Transaction;
```

Reply:
I will keep the inconsistency with receipt in mind and look into this again as I will work throughout transactions and receipts.

---

There is added benefit of it being a local type.

Local types can have foreign traits implemented for them, such as `From` trait from foreign type.

The local type does not automatically inherit traits for its inner type that it wraps, instead it supports only the behavior that we intentionally mean for it to have.

Any overhead gets optimized away thanks to zero-cost abstractions.